resources/views/admin/pages/category/index.blade.php

```diff
@@ -0,0 +1,94 @@
+                        <form>
+                            <div class="input-group">
+                                <input type="text" class="form-control bg-light small" placeholder="{{ trans('message.search') }}..." name="search" value="{{ \Request::get('search') }}">
+                                <div class="input-group-append">
+                                    <button class="btn btn-primary" type="submit">
+                                        <i class="fas fa-search fa-sm"></i>
+                                    </button>
+                                </div>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+            <div class="card-header py-3">
+                @if (Session::has('flash_message'))
+                <div class="alert alert-{{ Session::get('type_message') }}">
+                    {{ Session::get('flash_message') }}
+                </div>
+                @endif
+            </div>
+            <div class="card-body">
+                <div class="table-responsive">
+                    <table class="table table-bordered" id="dataTable" width="100%" cellspacing="0">
+                        <thead>
+                            <tr>
+                                <th>#</th>
+                                <th>{{ trans('message.category_name') }}</th>
+                                <th>{{ trans('message.parent_category') }}</th>
+                                <th>{{ trans('message.created_at') }}</th>
+                                <th>{{ trans('message.updated_at') }}</th>
+                                <th colspan="2" class="align">{{ trans('message.action') }}</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @foreach($categories as $key => $category)
+                            <tr>
+                                <td>{{ $key+1 }}</td>
+                                <td>{{ $category->name }}</td>
+                                <td>
+                                    @foreach ($parent_id as $key => $value)
+                                        @if ($value->id == $category->parent_id)
+                                            {{ $value->name }}
+                                        @endif
+                                    @endforeach
+                                </td>
+                                <td>
+                                    <?php
+                                        echo \Carbon\Carbon::createFromTimeStamp(strtotime($category->created_at))->diffForHumans();
+                                    ?>
+                                </td>
+                                <td>
+                                    <?php
+                                        echo \Carbon\Carbon::createFromTimeStamp(strtotime($category->updated_at))->diffForHumans();
```

Comment:
time này lấy ra mà sao phải dùng echo? 
Carbon là gì?

Reply:
1. Phải echo vì hàm sau chỉ xử lý thời gian chứ nó không in ra màn hình. Nếu không echo thì nó sẽ không được hiển thị
2. Carbon là 1 thư viện giúp xử lý thời gian.